remix-lib/src/universalDapp.js

```diff
@@ -241,6 +241,20 @@ module.exports = class UniversalDApp {
     })
   }
 
+   /**
+    * call the current given contract
+    *
+    * @param {String} to    - address of the contract to call.
+    * @param {String} data    - data to send with the transaction ( return of txFormat.buildData(...) ).
+    * @param {Function} callback    - callback.
+    */
+  sendRawTransaction (to, data, confirmationCb, continueCb, promptCb, callback) {
+    this.runTx({to, data, useCall: false}, confirmationCb, continueCb, promptCb, (error, txResult) => {
```

Comment:
Should `value` field also be there?

Reply:
`value` is handled by https://github.com/ethereum/remix-ide/blob/1ff9e8ce2bf553bb9ff2b1c6f3dc383f70806e28/src/app/udapp/run-tab.js#L57
to be refactored soon

Reply:
ok